services/frontend/domain/usecases/controller_validator.go

```diff
@@ -0,0 +1,71 @@
+package usecases
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	crlog "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/berops/claudie/internal/manifest"
+)
+
+// secretValidator validates Secrets containing the input-manifest
+type SecretValidator struct{}
+
+// validate takes the context and a kubernetes object as a parameter.
+// It will extract the secret data out of the received obj and run manifest validation against it
+func (v *SecretValidator) validate(ctx context.Context, obj runtime.Object) error {
+	log := crlog.FromContext(ctx)
+
+	secret, ok := obj.(*corev1.Secret)
+	if !ok {
+		return fmt.Errorf("expected a Secret but got a %T", obj)
+	}
+
+	log.Info("validating secret", "name", secret.Name)
```

Comment:
What is difference between `log.Info()` and `log.V(1).Info()` ? Also, I am not sure how Claudie can set the verbosity of this log. (we don't have `--v=5` flag)

Also, can we capitalise the logs, i.e. `Validating secret`

Reply:
The `log.V(1)` was from the boilerplate I generated, its the `logr` functionality to setup logging in more granular way. I will change it `log.Info()` as I missed out that one 👍